cmd/cloud_sql_proxy/proxy.go

```diff
@@ -328,16 +328,21 @@ func parseInstanceConfigs(dir string, instances []string, cl *http.Client) ([]in
 
 // CreateInstanceConfigs verifies that the parameters passed to it are valid
 // for the proxy for the platform and system and then returns a slice of valid
-// instanceConfig.
+// instanceConfig. It is possible for the instanceConfig to be empty if no valid
+// configurations were specified, however `err` will be set.
 func CreateInstanceConfigs(dir string, useFuse bool, instances []string, instancesSrc string, cl *http.Client) ([]instanceConfig, error) {
+	// Used to signify that there was no valid parsed config
+	var emptyConfigs []instanceConfig
+
 	if useFuse && !fuse.Supported() {
-		return nil, errors.New("FUSE not supported on this system")
+		return emptyConfigs, errors.New("FUSE not supported on this system")
```

Comment:
I thought an empty slice would be nicer than `nil`, though I am not sure what the "Go" way is. To be honest, I did this before I found `len` of a nil array was 0.

Is it more golang-like to revert this to using `nil`? I'm new to Golang.

Reply:
Thanks.